ml_peg/app/utils/utils.py

```diff
@@ -201,3 +212,170 @@ def get_scores(
         return raw_rows
 
     return scored_rows
+
+
+def _base_column_label(column: dict[str, Any]) -> str:
```

Comment:
Not especially important, but in terms of consistency, is there a particular reason behind the `_` for some of these functions, but not others?

Generally it's useful to indicate private functions, but none of this is really user-facing, so I'm not sure it's necessary.